apps/system-e2e/cypress.env.json

```diff
@@ -2,19 +2,33 @@
   "fakeUsers": [
     {
       "name": "María Sól ÞÍ Torp",
-      "phoneNumber": "0101450"
+      "phoneNumber": "0101450",
+      "children": []
     },
     {
       "name": "Sayed Wahab ÞÍ Sverrisson",
-      "phoneNumber": "0101479"
+      "phoneNumber": "0101479",
+      "children": []
+    },
+    {
+      "name": "Gervimaður Afríka",
+      "phoneNumber": "0103019",
+      "nationalId": "0101303019",
+      "children": [
+        {
+          "name": "Sól Rún Gervimannsdóttir",
+          "nationalId": "1111111119"
+        }
+      ]
     }
   ],
+  "cognitoUrl": "https://cognito.shared.devland.is",
   "local": {
     "authUrl": "https://identity-server.dev01.devland.is",
     "baseUrl": "http://localhost:4200"
   },
   "dev": {
     "authUrl": "https://identity-server.dev01.devland.is",
-    "baseUrl": "https://beta.dev01.devland.is"
```

Comment:
This is going to be a problem I think

Reply:
using `https://loftbru.dev01.devland.is` now

Reply:
This will change the base url for all tests, right?

Reply:
yes, if I remember correctly we had an issue with hardcoded prod url in and this fixed.

See: https://github.com/island-is/island.is/issues/7906

Reply:
Let me confirm that this doesnt break other tests.

Reply:
reverted to beta.dev01